CSharpGeneratorFunctions.sh

```diff
@@ -0,0 +1,30 @@
+# This is intended to be imported using the "source" function
+# from any script that wants to invoke the C#-based generator.
+
+CSHARP_GENERATOR_DIR=tmp-gapic-generator-csharp
+
+# Installs the C# generator in a tmp-gapic-generator-csharp directory.
+# If the directory already exists, it is deleted first.
+install_csharp_generator() {
+  echo "Installing C# generator in $CSHARP_GENERATOR_DIR"
+  rm -rf $CSHARP_GENERATOR_DIR
+  
+  git clone --quiet --recursive --depth=1 \
+    https://github.com/googleapis/gapic-generator-csharp.git \
+    $CSHARP_GENERATOR_DIR
+    
+  echo 'Building C# generator'
+  (cd $CSHARP_GENERATOR_DIR && \
+   dotnet build -c Release -nologo -clp:NoSummary -v quiet Google.Api.Generator.Rest)
+  echo 'Generator build complete'
+}
+
+# Runs the C# generator, after ensuring it's installed.
```

Comment:
nit: This function doesn't appear to ensure it's installed.

Reply:
Whoops, you're right. Will fix, repush, then merge on green.